src/components/ResourceItem/README.md

```diff
@@ -0,0 +1,212 @@
+  />
+</Card>
+```
+
+### Item with media
+
+The media element can hold an [avatar](/components/images-and-icons/avatar), [thumbnail](/components/images-and-icons/thumbnail) or other small-format graphic.
+
+```jsx
+<Card>
+  <ResourceList
+    resourceName={{singular: 'customer', plural: 'customers'}}
+    items={[
+      {
+        id: 145,
+        url: 'customers/145',
+        avatarSource: 'https://avatars.io/twitter/Astro_Soyeon',
+        name: 'Yi So-Yeon',
+        location: 'Gwangju, South Korea',
+      },
+    ]}
+    renderItem={(item) => {
+      const {id, url, avatarSource, name, location} = item;
+
+      return (
+        <ResourceItem
+          id={id}
+          url={url}
+          media={
+            <Avatar customer size="medium" name={name} source={avatarSource} />
+          }
+          accessibilityLabel={`View details for ${name}`}
+        >
+          <h3>
+            <TextStyle variation="strong">{name}</TextStyle>
+          </h3>
+          <div>{location}</div>
+        </ResourceItem>
+      );
+    }}
+  />
+</Card>
+```
+
+### Item with shortcut actions
+
+Shortcut actions present popular actions from the resource’s details page for easy access. A shortcut action should available on every item in the list.
+
+```jsx
+<Card>
+  <ResourceList
+    resourceName={{singular: 'customer', plural: 'customers'}}
+    items={[
+      {
+        id: 145,
+        url: 'customers/145',
+        avatarSource: 'https://avatars.io/twitter/Astro_Soyeon',
+        name: 'Yi So-Yeon',
+        location: 'Gwangju, South Korea',
+        latestOrderUrl: 'orders/1456',
+      },
+    ]}
+    renderItem={(item) => {
+      const {id, url, avatarSource, name, location, latestOrderUrl} = item;
+      const shortcutActions = latestOrderUrl
+        ? [{content: 'View latest order', url: latestOrderUrl}]
+        : null;
+
+      return (
+        <ResourceItem
+          id={id}
+          url={url}
+          media={
+            <Avatar customer size="medium" name={name} source={avatarSource} />
+          }
+          shortcutActions={shortcutActions}
+          accessibilityLabel={`View details for ${name}`}
+        >
+          <h3>
+            <TextStyle variation="strong">{name}</TextStyle>
+          </h3>
+          <div>{location}</div>
+        </ResourceItem>
+      );
+    }}
+  />
+</Card>
+```
+---
+## Required components
+
+The resource item component must be wrapped in the [resource list](/components/lists-and-tables/resource-list) component.
+
+---
+
+## Accessibility
+
+Resource items function as links to the full-page representations of the items. Each item should have a unique `name` prop. For each `ResourceItem`, the `accessibilityLabel` prop should be used to give the link a unique `aria-label` value. The `aria-label` value should convey the link’s purpose, using the `name` value. Merchants who use screen readers or other text to speech tools should be able to easily distinguish each link from the others.
```

Comment:
Not super clear what this means. "Function as" or "are"?

Reply:
Maybe swop out "should" to make it more active, eg "Use the aria-label to convey the link's purpose"
(optional)